cukedoctor-multipage-layout/src/main/java/cukedoctor/multipage/defaults/DefaultMultipagePersister.java

```diff
@@ -0,0 +1,39 @@
+package cukedoctor.multipage.defaults;
+
+import com.github.cukedoctor.api.CukedoctorDocumentBuilder;
+import com.github.cukedoctor.api.DocumentAttributes;
+import com.github.cukedoctor.config.CukedoctorConfig;
+import com.github.cukedoctor.i18n.I18nLoader;
+import com.github.cukedoctor.util.FileUtil;
+import cukedoctor.multipage.model.MultipageDocumentation;
+import cukedoctor.multipage.model.Page;
+import cukedoctor.multipage.spi.MultipagePersister;
+
+public class DefaultMultipagePersister implements MultipagePersister {
+    @Override
+    public void persist(MultipageDocumentation multipageDocumentation, String outputFolderLocation) {
+        for(Page page: multipageDocumentation.getPages()) {
+            FileUtil.saveFile(outputFolderLocation + page.getPageTitle() + ".adoc", page.getRender());
+        }
+    }
+
+    @Override
```

Comment:
These methods are not needed but I had to implement "BaseRenderer" in order to make the service loader work.

Reply:
Maybe we can create a `Renderer` marker interface without any method which BaseRenderer can extend, what do you think?

Reply:
I think they are needed because (1) you intend to instantiate the object via ServiceLoaderUtil (I think) and (2) some are needed by child objects (e.g. the renderers).